docs/setup.rst

```diff
@@ -118,7 +118,9 @@ file.
    sudo -u redash bin/run ./manage.py org set_google_apps_domains {{domains}}
 
 
-If you're passing multiple domains, separate them with commas.
+If you're passing multiple domains, separate them with commas. 
+
+These are the domains you will allow users to login from. I.e. `gmail.com` will allow all users trying to login with a gmail.com address to login. Other addresses will not be allowed. 
```

Comment:
Thanks for submitting an update for the docs. This is always appreciated :-) How about this alternative for the phrasing:

These are the (Google Apps) domains you will allow users to create accounts with; i.e. `redash.io` will allow all users trying to create an account with a `@redash.io` Google account to create an account and login. Other addresses will not be allowed. 

(Main change is to use a non Gmail domain and some clarification that we talk about Google accounts)